File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/AnoncredsAccumulatorFullException.java
package org.hyperledger.indy.sdk;

public class AnoncredsAccumulatorFullException extends IndyException
{
	private final static String message = "The anoncreds accumulator is full.";

	AnoncredsAccumulatorFullException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/AnonCredsApi/AnoncredsAccumulatorFullException.cs
namespace Hyperledger.Indy.AnonCredsApi
{
    /// <summary>
    /// Exception thrown when an anoncreds accumulator is full.
    /// </summary>
    public class AnoncredsAccumulatorFullException : IndyException
    {
        const string message = "The anoncreds accumulator is full.";

        /// <summary>
        /// Initializes a new AnoncredsAccumulatorFullException.
        /// </summary>
        internal AnoncredsAccumulatorFullException() : base(message, (int)ErrorCode.AnoncredsAccumulatorIsFull)
        {

        }
    }

}
